railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -116,15 +128,18 @@
 <% end %>
 
 <script>
-  function formatChanged(form) {
-    var part_name = form.options[form.selectedIndex].value
-    var iframe =document.getElementsByName('messageBody')[0];
-    iframe.contentWindow.location.replace(part_name);
+  function refreshBody() {
+    var part_select = document.querySelector('select#part');
+    var locale_select = document.querySelector('select#locale');
+    var part_param = part_select.options[part_select.selectedIndex].value;
+    var locale_param = locale_select.options[locale_select.selectedIndex].value;
```

Comment:
`locale_select` only appears in `I18n.available_locales.count > 1`. 
So if users have only one locale, this code will occur error.